pkg/render/intrusion_detection.go

```diff
@@ -532,7 +534,7 @@ func (c *intrusionDetectionComponent) intrusionDetectionControllerContainer() co
 		},
 	}
 
-	privileged := false
+	sc := securitycontext.NewBaseContext(securitycontext.RunAsUserID, securitycontext.RunAsGroupID)
```

Comment:
Nothing is set in IDS controller `Dockerfile` so we are safe to set UID and GID to 10001.